### ISSUE TITLE
Header navigation alignment and Login button colour

### ISSUE THREAD

**Issue by Android63** [NHSLeadership/nightingale-wp#95]
Address points raised by @Pro-Paul at https://github.com/NHSLeadership/nhsl-wp-ldphub/issues/13

**Comment by Android63** [comment]
![image](https://user-images.githubusercontent.com/1991226/35331089-0e82e64a-00fe-11e8-9d4e-584059614ca8.png)

Following discussion with @Pro-Paul, here are the requirements for desktop layout of the header:

- The header should have padding of 30px, apart from the bottom, which should be 15px.

- The logo, which could be of any size, should be left-aligned in a 300px column.

- The nav column, which may contain a search bar, menu bar and login button, should be 632px wide.

- There should be a 32px horizontal gutter between the logo column and the nav column.

- The top 32px of the nav column should be reserved for a search bar which, if present, should occupy 100% of its column width.

- The menu bar, if present, should have a height of 19px and should be left-aligned in its column with a 15px top margin.

- This should result in the menu items vertically aligning with the first line of text in the logo. Since the logo is (currently) an image, this will be impossible to control in markup.

- The login button, if present, should be 84px by 36px, should be right-aligned in its column with a 10px top margin.

**Comment by cehwitham** [comment]
Presumably, this is markup related @Android63? 
If the markup matches Nightingale, then the layout should too?

**Comment by Android63** [comment]
Yes indeed @cehwitham. Awaiting https://github.com/NHSLeadership/nightingale/pull/185

**Comment by Android63** [comment]
Since https://github.com/NHSLeadership/nightingale/pull/185 is now complete, this issue can be addressed.
